sdk/PrebidServerAdapter/PBServerAdapter.m

```diff
@@ -103,121 +114,110 @@ - (NSURLRequest *)buildRequestForAdUnits:(NSArray<PBAdUnit *> *)adUnits {
     }
 }
 
-- (NSDictionary *)requestBodyForAdUnits:(NSArray<PBAdUnit *> *)adUnits {
+- (NSDictionary *)openRTBRequestBodyForAdUnits:(NSArray<PBAdUnit *> *)adUnits {
     NSMutableDictionary *requestDict = [[NSMutableDictionary alloc] init];
 
-    if (self.primaryAdServer == PBPrimaryAdServerMoPub || self.primaryAdServer == PBPrimaryAdServerUnknown) {
-        requestDict[@"cache_markup"] = @(1);
-    }
+    requestDict[@"id"] = @"some-request-id";
+    requestDict[@"test"] = @(1);
```

Comment:
remove test :1

Reply:
This is under debug macro... will automatically be removed when turned to release mode